src/main/java/io/excaliburfrc/robot/subsystems/Climber.java

```diff
@@ -11,20 +22,54 @@
 import java.util.function.DoubleSupplier;
 
 public class Climber extends SubsystemBase implements AutoCloseable {
-  private final DoubleSolenoid anglerPiston =
+  private final DoubleSolenoid piston =
       new DoubleSolenoid(
           PneumaticsModuleType.CTREPCM,
           ClimberConstants.FORWARD_CHANNEL,
           ClimberConstants.REVERSE_CHANNEL);
-  private final CANSparkMax motor =
-      new CANSparkMax(ClimberConstants.CLIMBER_SPARKMAX, MotorType.kBrushless);
-  private final DigitalInput sensor = new DigitalInput(ClimberConstants.SENSOR_CHANNEL);
+  private final CANSparkMax leftMotor =
+      new CANSparkMax(ClimberConstants.LEFT_MOTOR_ID, MotorType.kBrushless);
+  private final CANSparkMax rightMotor =
+      new CANSparkMax(ClimberConstants.RIGHT_MOTOR_ID, MotorType.kBrushless);
+  private final RelativeEncoder leftEncoder = leftMotor.getEncoder();
+  private final SparkMaxPIDController leftController = leftMotor.getPIDController();
+  private final RelativeEncoder rightEncoder = rightMotor.getEncoder();
+  private final SparkMaxPIDController rightController = rightMotor.getPIDController();
+
+  private final ElevatorFeedforward feedforward =
+      new ElevatorFeedforward(0, MG * Math.cos(ANGLE), kV, kA);
```

Comment:
the weight is ~constant. the output is what should be multiplied by the angle's sin/cos

Reply:
The output includes friction and stuff that the angle doesn't change